Tasks/UserInterface/00_UserInterface.md

```diff
@@ -0,0 +1,32 @@
+# User interface
+
```

Comment:
Prüf noch mal die Links, bei mir gehen alle auf der dritten Ebene nicht

Reply:
Bei mir in GitHub auch nicht. In VSC funktionieren.
Es war auch so in Warehousing und sie funktionieren jetzt auf der Webseite, also tippe ich auf ein Problem in GitHub.